package-lock.json

```diff
@@ -2120,17 +2120,17 @@
       "integrity": "sha512-x5HFsW+E/nQalGMw7hu+fvPqnBeBaIr0lWJ2SG0PPL2j+Pm9lYvCrsZJGIgauPIENx0v10INIyFjmSNUD/gSqQ=="
     },
     "@babel/plugin-external-helpers": {
-      "version": "7.8.3",
```

Comment:
Note to self: I think this will break mobile again. Try to avoid such a massive change in package-lock before merge.